src/vscode-dts/vscode.proposed.notebookControllerAffinityHidden.d.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+	// https://github.com/microsoft/vscode/issues/161144
+	export enum NotebookControllerAffinityExtended {
```

Comment:
I was a slight bit unsure on the best way to just extend an enum, didn't see an easy "typescripty" way to add in the new value. So I went with just adding a new extended type and updating the controller function to take either type. Would love to know if there is a better way to do this.

Reply:
This is fair - there are limits to what we can do with TypeScript. On a nit-note: in such cases we usually use the `2` suffix, not `Extended`, so `NotebookControllerAffinity2` might be a bit better